fastjet/internal/delaunay/point.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2017 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package delaunay
+
+import (
+	"math"
+	"strconv"
+)
+
+// Point in the X-Y Plane, that holds dynamic information
+type Point struct {
+	X       float64     // X coordinate
+	Y       float64     // Y coordinate
+	adjT    []*Triangle // holds all triangles adjacent to a point
+	outside bool        // true if point is on the outside of the triangulation
+	Ind     int         // user defined identifier
```

Comment:
I apologize, I forgot to mention that I need it in inPolygon. And yes that is mostly true. The points on the polygon are indexed in order. If there are 4 points around the point to be removed, they are indexed counterclockwise. Triangle with index 4,1,2 is inside the polygon, just like 1,2,3 or 3,4,1 for example. But Triangle 1,3,2 would be outside the polygon. That works, because A,B,C are ordered counterclockwise